SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -104,7 +114,7 @@ void ComponentReader::ApplySchemaObject(Schema_Object* ComponentObject, UObject*
 
 			uint8* Data = (uint8*)Object + SwappedCmd.Offset;
 
-			if (bIsInitialData || GetPropertyCount(ComponentObject, FieldId, Cmd.Property) > 0 || ClearedIds->Find(FieldId) != INDEX_NONE)
+			if (bIsInitialData || GetPropertyCount(ComponentObject, FieldId, Cmd.Property) > 0 || ClearedIds.Find(FieldId) != INDEX_NONE)
```

Comment:
Is this ever false? It seems that with the new logic, every ID we iterate through will pass this check, so we should remove it completely. It would also mean we don't have to pass `bIsInitialData` and `ClearedIds`.